src-plugins/manualRegistration/manualRegistration.cpp

```diff
@@ -41,14 +32,16 @@ class manualRegistrationPrivate
 {
 public:
     manualRegistration * proc;
-    template <class PixelType>
-    int update();
-    template < typename TFixedImage, typename TMovingImage >
-    bool write(const QString&);
-    void * registrationMethod ;
+    template <class PixelType> int update();
+    template <typename PixelType, typename TransformType> int applyRegistration();
+
     QList<manualRegistrationLandmark*> * FixedLandmarks;
     QList<manualRegistrationLandmark*> * MovingLandmarks;
-    TransformType::Pointer transform;
+
+    // Transformation methods
+    TransformType_Rigid3D::Pointer transform_Rigid3D;
+    TransformType_Affine::Pointer  transform_Affine;
```

Comment:
I think these two variables could become one by using the mother class of these transforms: itk::MatrixOffsetTransformBase.
Using an instance of this class would simplify the code.